templates/repo/commit_statuses.tmpl

```diff
@@ -1,6 +1,14 @@
-{{if eq (len .Statuses) 1}}{{$status := index .Statuses 0}}{{if $status.TargetURL}}<a class="ui link commit-statuses-trigger gt-vm" href="{{$status.TargetURL}}">{{template "repo/commit_status" .Status}}</a>{{end}}{{end}}
-<div class="ui commit-statuses-popup commit-statuses tippy-target">
-	<div class="ui relaxed list divided">
+{{if .Statuses}}
+	{{if and (eq (len .Statuses) 1) .Status.TargetURL}}
```

Comment:
`if and`?
Leftover from earlier?

Reply:
That’s how go template logic works.

`if and p1 p2` means `p1 && p2`

Reply:
Ah, I have written to much Python code to easily grasp it. Fair enough.

Reply:
To be fair this syntax is far from the expected “go simplicity” ;)